docs/rep/README.md

```diff
@@ -1 +1,64 @@
-## RIK Enhancement Proposals
+
+# RIK Enhancement Proposals
+
+  
+  
+  
+  
+
+# Controller
+
+It allows to handle state of cluster by interracting with CLI and scheduler
+
+  
+![controler schema](https://res.cloudinary.com/malo-polese/image/upload/v1617964829/Polytech_DO/rik/controller_c3ood6.png)
+### External API (with CLI)
+
+REST API
+
+Can receive YAML file for cluster, deployment, pods, configurations the controller has to check that the given rules are compliant and gives instructions to scheduler accordingly.
+
+We may use [Rocket](https://rocket.rs/) as a web server framework.
+
+#### Endpoints :
+
+
+
+TBD
```

Comment:
A good starting point would be endpoints meeting the RIK CLI command proposal at #4

Reply:
We have added some external endpoints.

Reply:
Nevermind we will update them to use a HTTP RPC shape and OpenAPI3 specification.

Reply:
Done.  
The tenant part is not really complete as we are not sure how we want to handle that part.